packages/web/src/apollo/index.tsx

```diff
@@ -50,7 +50,7 @@ export type {
 
 export { useCache }
 
-export { fragmentRegistry, registerFragment, registerFragments, SSELink }
+export { fragmentRegistry, registerFragment, registerFragments }
```

Comment:
@dthyresson is there a reason this as re-exported?

Reply:
The can be used as in

![image](https://github.com/redwoodjs/redwood/assets/1051633/c75f191b-9205-446d-b462-2edf9bdbe6a3)

Reply:
See docs in https://github.com/redwoodjs/redwood/pull/9140

Reply:
@dthyresson sorry should've been more specific, I meant `SSELink`—it was re exported

Reply:
Ah, no - SSELink does not need to be exported. Thanks!